### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matthiashimmelmann**
Added documentation and short description to `Readme.md`

### CODE REVIEW

**Comment by Legersky** [comment]
When I click the MIT badge, I get a blank page containing only that badge.

**Comment by matthiashimmelmann** [comment]
> When I click the MIT badge, I get a blank page containing only that badge.

True, that's probably not ideal.

**Comment by Legersky** [comment]
I would link our LICENSE file.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
 <p align="center">
 <img src="assets/icon.jpg" width="260">
 </p>
+
+[![General badge](https://img.shields.io/badge/PyRigi-Documentation-blue?style=plastic&link=pyrigi.github.io%2FPyRigi%2F%20)](https://pyrigi.github.io/PyRigi/) [![General badge](https://img.shields.io/badge/license-MIT-yellow?style=plastic)](https://en.wikipedia.org/wiki/MIT_License)
```
Comment (MatteoGallet) [inline README.md:5]: Here I would not link the Wikipedia page for the license, I would rather link to the specific license we use (i.e., the file LICENSE).

**Review by MatteoGallet — APPROVED** [review]

**Pull request merged**